### PR TITLE
Blame Command

### PR DESCRIPTION
**Pull request by LostNTime**
This command will generate one of those blame someone emotes that we are constantly using on the server.

### CODE REVIEW

File: commands/social/blame.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		
+		const canvas = new Canvas(300, 300);
+		const ctx = canvas.getContext('2d');
+
+		const lines = this.wrapText(ctx, personalMessage, 110);
+
+		const base = new Image();
+		
+
+		const generate = () => {
```
Comment (TeeSeal) [inline commands/social/blame.js]: Declare outside of run method and pass ctx as parameter.

Comment (iCrawl) [inline commands/social/blame.js]: ? This doesn't seem necessary.

Comment (TeeSeal) [inline commands/social/blame.js]: Just thought why declare one for each command run if you can have one for all runs ¯\_(ツ)_/¯

Comment (iCrawl) [inline commands/social/blame.js]: Because you kinda need to regenerate it? lol

Comment (TeeSeal) [inline commands/social/blame.js]: could return ctx at the end, but I'm probs missing something, never worked with canvas yet. If you think it's alright this way, then so be it

**Comment by iCrawl** [comment]
I updated it a bit, but I still would need the image you are using for base.src

**Comment by LostNTime** [comment]
@iCrawl I will add it once I get back home tomorrow.

**Comment by LostNTime** [comment]
ok the background is done.

**Comment by alucard0134** [comment]
4 commits later

**Comment by LostNTime** [comment]
@iCrawl can you merge this already?

**Comment by alucard0134** [comment]
@vzwGrey

**Pull request merged**